src/template.html

```diff
@@ -14,5 +14,6 @@
 <body>
 	<div id='app'>%sapper.html%</div>
 	%sapper.scripts%
+	<script src="https://rawgit.com/Microsoft/TypeScript/master/lib/typescriptServices.js"></script>
```

Comment:
I think we can use `https://raw.githubusercontent.com/microsoft/TypeScript/master/lib/typescriptServices.js` as rawgit has reached EOL in oct 2018, to avoid surprises.

Reply:
I also have to revise if there's a solution somewhere on including the Typescript compiler in a Sapper/Svelte build and how to deal with the bundling times I experienced (more than ten minutes!).

Reply:
https://github.com/sveltejs/svelte-preprocess#my-typescript-compilation-is-sloooooooow
https://github.com/sveltejs/sapper/pull/1222
https://discordapp.com/channels/457912077277855764/457912077277855766/722838119950319724